### DISCUSSION TITLE
how do you handle nested edit grids?

### DISCUSSION THREAD

**Discussion by bulivlad** [formio/formio.js#4880] [Q&A] [answered]
hi,
is there any way to get the parent row values in a nested grids setup? i ended up with 2 level nested grids in my form and i need to populate a field in a child grid based on a value picked in the parent grid. when retrieving data from the same row in the grid i can use `row` object but is there any `parentRow` or a similar way to get data from the parent?

the form setup looks something like 
![IMG_0194](https://user-images.githubusercontent.com/6691306/192832117-3d023ec7-99fc-4733-b518-e56eb251a6f2.jpeg)

the form json below
`{
  "display": "form",
  "components": [
    {
      "label": "Parent edit grid",
      "tableView": false,
      "rowDrafts": false,
      "key": "parentGrid",
      "type": "editgrid",
      "displayAsTable": false,
      "input": true,
      "components": [
        {
          "label": "Parent select",
          "widget": "choicesjs",
          "tableView": true,
          "data": {
            "values": [
              {
                "label": "yes",
                "value": "yes"
              },
              {
                "label": "no",
                "value": "no"
              },
              {
                "label": "idk",
                "value": "idk"
              }
            ]
          },
          "key": "parentSelect",
          "type": "select",
          "input": true
        },
        {
          "label": "Child data grid",
          "reorder": false,
          "addAnotherPosition": "bottom",
          "layoutFixed": false,
          "enableRowGroups": false,
          "initEmpty": false,
          "tableView": false,
          "defaultValue": [
            {}
          ],
          "key": "childDataGrid",
          "type": "datagrid",
          "input": true,
          "components": [
            {
              "label": "Child select",
              "widget": "choicesjs",
              "tableView": true,
              "key": "childSelect",
              "type": "select",
              "input": true
            }
          ]
        }
      ]
    },
    {
      "type": "button",
      "label": "Submit",
      "key": "submit",
      "disableOnInvalid": true,
      "input": true,
      "tableView": false
    }
  ]
}`

**Comment by travist** [comment, marked as answer]
You would probably need to use the "instance" property to get that.

```js
// "instance" points to Child Select, "instance.parent" points to Child data grid, "instance.parent.data" points to the "row" data for the Child data grid.
instance.parent.data
```
